build-scripts/build.sh

```diff
@@ -1,10 +1,59 @@
 #!/usr/bin/env bash
-set -e
 
+postTheme2Actions()
+{
+     if [ $? -eq 0 ]
+            then
+                node cypress/support/createReports.js
+                zip -r output/reports/screenshots.zip cypress/screenshots
+                exit 0
+            else
+                echo "The Cypress tests failed - Uploading report to S3 with no latest" >&2
+                node cypress/support/createReports.js
+                zip -r output/reports/screenshots.zip cypress/screenshots
+                $UPLOADTOS3 nolatest
+                exit 1
+        fi
+}
+
+postTheme1Actions()
+{
+   if [ $? -eq 0 ]
+            then
+                echo "The Cypress tests failed - Uploading report to S3 with no latest" >&2
+                node cypress/support/createReports.js
+                zip -r output/reports/screenshots.zip cypress/screenshots
+                $UPLOADTOS3 nolatest
+                exit 1
+            else
+                echo "The Cypress tests failed - Uploading report to S3 with no latest" >&2
+                node cypress/support/createReports.js
+                zip -r output/reports/screenshots.zip cypress/screenshots
+                $UPLOADTOS3 nolatest
+                exit 1
+        fi  
+}
+
+set -e 
+source /etc/profile
+nvm install 12.19.0
```

Comment:
we do this in Jenkins too, is there a reason its also in here

Reply:
Because its in the shell script, line 38 is needed to it has access to the S3 upload file. 
The version of npm was also getting reset as a result of line 38 back to the default version Jenkins used so we needed to set it again to make sure it uses the correct version